templates/satspay/display.html

```diff
@@ -221,6 +292,56 @@
 <script>
   Vue.component(VueQrcode.name, VueQrcode)
 
+  Vue.component('satspay-paid',
```

Comment:
why not have the components in their own files?

Reply:
I'm just now learning how to do components... may do that further ahead!